### PR TITLE
Add files via upload

### LINKED ISSUE #114: Make a setup.bat file for windows
Check out the [setup.sh](https://github.com/CDC-IITDH/cdc-placement-website-backend/blob/main/setup.sh) file and do all those commands to run in windows

### PR DESCRIPTION
**Pull request by kasv-p**
place superuser.bat file in cdc_backend folder and setup.bat in cdc folder.
updated setup.sh
please check and review

closes #114

### CODE REVIEW

**Comment by CDC-IITDH** [comment]
Can you merge both `setup.bat` and `superuser.bat`. Like make it one file `setup.bat`

**Review by gowtham3105 — DISMISSED** [review]
Merge both files

File: setup.bat [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+start /wait  python -m venv venv && venv\Scripts\activate && pip install -r requirements.txt &^
+echo Environment Setup Complete &^
+timeout 3 > NUL &^
+cd "CDC_Backend" &^
+
+python manage.py flush --no-input &^
+python manage.py makemigrations &^
+python manage.py migrate &^
+echo Migrations complete &^
+start superuser.bat &^
```
Comment (karthikmurakonda) [inline setup.bat:11]: add this line:
`cd .. &^`
change the line:
`start .\superuser.bat &^`

File: superuser.bat [hunk 2]
```diff
@@ -0,0 +1,11 @@
+@echo off
+set /p create="do you want to create supruser ? (Y/N) "
+
```
Comment (karthikmurakonda) [inline superuser.bat:3]: add this line: `cd "CDC_Backend" &^`

**Review by karthikmurakonda — CHANGES_REQUESTED** [review]

**Review by karthikmurakonda — CHANGES_REQUESTED** [review]
.

**Comment by karthikmurakonda** [comment]
closes #114

**Comment by gowtham3105** [comment]
Try to finish this by tomorrow

**Review by karthikmurakonda — APPROVED** [review]

**Pull request merged**